frames/src/main/java/com/checkout/frames/cvvinputfield/api/InternalCVVComponentMediator.kt

```diff
@@ -16,11 +16,15 @@ internal class InternalCVVComponentMediator(
     private val publicKey: String,
     private val environment: Environment,
     private val context: Context,
+    var isCVVComponentCalled: Boolean = false,
```

Comment:
Shall we make this a local variable instead of an param if there's no specific reason of doing so?

Reply:
Fair point. However, this value helps in UI testing for the component (InternalCVVComponentMediatorTest) and the class is internal.

Reply:
Sure done